superset/connectors/sqla/models.py

```diff
@@ -484,30 +487,34 @@ def get_from_clause(self, template_processor=None, db_engine_spec=None):
             return TextAsFrom(sa.text(from_sql), []).alias('expr_qry')
         return self.get_sqla_table()
 
-    def adhoc_metric_to_sa(self, metric, cols):
+    def adhoc_metric_to_sa(self, metric, cols, db_engine_spec):
```

Comment:
Out of scope for this PR, but I just caught the `..._to_sa` should be `..._to_sqla` to be consistent with aliasing sqla.

Reply:
Renamed both method and variables from `sa` to `sqla`.